src/components/SecretsModal/UniversalFields.js

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { Select, SelectItem, TextInput } from 'carbon-components-react';
+import './SecretsModal.scss';
+
+const UniversalFields = props => {
+  const {
+    name,
+    handleChange,
+    accessTo,
+    selectedNamespace,
+    namespaces,
+    invalidFields
+  } = props;
+
+  return (
+    <>
+      <TextInput
+        id="name"
+        data-testid="name"
+        placeholder="secret-name"
+        value={name}
+        labelText="Name:"
+        onChange={handleChange}
+        invalid={invalidFields.indexOf('name') > -1}
+        invalidText="Must be less than 563 characters, contain only lowercase alphanumeric character, . or -"
+        autoComplete="off"
+      />
+      <br />
+      <Select
```

Comment:
We have a NamespacesDropdown component to ensure consistency

Reply:
ill update it to use that dropdown